src/test/beatmaptest.cpp

```diff
@@ -10,6 +10,15 @@ using namespace mixxx;
 
 namespace {
 
+int countBeatsInIterator(std::unique_ptr<BeatIterator> pIterator) {
```

Comment:
An iterator is a pointer tow a beat. So there can be now beats in it. 
numberOfRemainingBeats() or such might be fit better. 
Or numberOfBeatsinRange() ..

Reply:
The iterator is created by std::lower_bound
"Returns an iterator pointing to the first element in the range [first, last]"

Reply:
https://github.com/mixxxdj/mixxx/pull/4095/commits/2d47336b7589c7524afd797f6945fd7d87faaa76